config/artifact-caching-proxy__common.yaml

```diff
@@ -1,11 +1,11 @@
 # Common values for all nginx-proxies
 resources:
   limits:
-    cpu: 2
-    memory: 1024Mi
+    cpu: 3
+    memory: 6146Mi
```

Comment:
Will it work on the 3 clusters? (e.g. do you mind adding links to the 3 node size definition in the PR body)

Reply:
I've added the node capacity on the 3 clusters.